source/features/select-notifications.tsx

```diff
@@ -85,6 +89,7 @@ function createDropdownList(category: Category, filters: Filter[]): JSX.Element
 		'Pull requests': <GitPullRequestIcon className="color-fg-muted"/>,
 		Issues: <IssueOpenedIcon className="color-fg-muted"/>,
 		Open: <CheckCircleIcon className="color-fg-success"/>,
+		Others: <SquirrelIcon className="color-fg-muted"/>,
```

Comment:
https://primer.style/foundations/icons/squirrel-16

Alternatives: https://primer.style/foundations/icons#16px

Reply:
Suggestions welcome even after the PR is merged

Reply:
Some ideas:
- https://primer.style/foundations/icons/apps-16
- https://primer.style/foundations/icons/stack-16
- https://primer.style/foundations/icons/unread-16
- https://primer.style/foundations/icons/filter-remove-16
- https://primer.style/foundations/icons/rss-16